### ISSUE TITLE
Warning, will become error in Angular 5.x

### ISSUE THREAD

**Issue by arsyed04** [ng-bootstrap/ng-bootstrap#1849]
During angular-cli AOT build process:

12% building modules 21/25 modules 4 active ...e\release\assets\fonts\data-table.svg
Warning: Can't resolve all parameters for NgbModalRef in /node_modules//modal/modal-ref.d.ts: ([object Object], [object Object], [object Object], ?). This will become an error in Angular v5.x
Warning: Can't resolve all parameters for NgbModalRef in /node_modules//modal/modal-ref.d.ts: ([object Object], [object Object], [object Object], ?). This will become an error in Angular v5.x

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 4.3.6
Angular-Cli: 1.4.1
ng-bootstrap: 1.0.0-beta.4
Bootstrap: 4.0.0-beta

**Comment by pkozlowski-opensource** [comment]
See: https://github.com/ng-bootstrap/ng-bootstrap/issues/1840#issuecomment-328529567

tl;dr; already fixed in master, will be part of the next release.

**Comment by pkozlowski-opensource** [comment]
FYI, just released 1.0.0-beta.5 that removes the mentioned warning.
